.github/workflows/release.yml

```
@@ -101,35 +45,48 @@ jobs:
+        if: steps.release_policy.outputs.release_enabled == 'true'
         run: git tag --list '@better-translate/*@*' | sort > "$RUNNER_TEMP/release-tags-before.txt"
 
-      - name: Publish packages from main
+      - name: Create release pull request or publish packages
+        if: steps.release_policy.outputs.release_enabled == 'true'
+        id: changesets
+        uses: changesets/action@v1
+        with:
+          version: bun run ci:version
+          publish: bun run ci:release
+          title: chore: release packages
```

actionlint (1.7.12)

[error] could not parse as YAML: mapping values are not allowed in this context

(syntax-check)